app/assets/javascripts/scp-igv.js

```diff
@@ -140,15 +140,38 @@ function initializeIgv() {
   const igvContainer = document.getElementById('igv-container')
 
   const genes = $('.queried-gene')
-  const locus = (genes.length === 0) ? ['myc'] : [genes.first().text()]
 
-  const genome = bamsToViewInIgv[0].genomeAssembly
+  const genomeId = bamsToViewInIgv[0].genomeAssembly
+
+  let reference
+  let locus
+  if (genomeId === 'Macaca_fascicularis_5.0') {
```

Comment:
this is nice -- I like how clearly this is hardcoded, rather than half-abstracted, given that this is the only kind of this assembly we have now.  This makes both this easier to read now, and hopefully will make subsequent refactoring to support more of these kinds of assemblies easier.